src/Maestro/maestro-angular/src/app/page/build/build.component.ts

```diff
@@ -59,12 +60,15 @@ export class BuildComponent implements OnInit, OnChanges {
   public toastDate?: Date;
   public acceptToast?: () => void;
 
+  public includeSubToolsets: boolean = false;
```

Comment:
How is this different from the includeToolsets variable above? Does subscription need its own toolsets toggle?

Reply:
That is an excellent question best answered by someone who will actually be using it. :) I'll follow up on this.

Reply:
@mmitche Can you weigh in on this? Are toolsets an important part of this view and should they be toggled separately from the dependency view?

Reply:
Resolving this comment since it was split into a separate issue & pulled out of this PR.